libnmstate/nm/connection.py

```diff
@@ -387,20 +387,26 @@ def _delete_connection_callback(self, src_object, result, user_data):
     @staticmethod
     def _commit_changes_callback(src_object, result, user_data):
         mainloop, nmdev = user_data
+        devname = src_object.get_interface_name()
         try:
             success = src_object.commit_changes_finish(result)
         except Exception as e:
             if mainloop.is_action_canceled(e):
                 logging.debug("Connection update aborted: error=%s", e)
             else:
+                if nmdev:
+                    iface_name = nmdev.props.interface
+                    iface_state = nmdev.props.state
+                else:
+                    iface_name = devname
+                    iface_state = "absent"
```

Comment:
Why `absent`?

Reply:
It is not there... the device.
Any other suggestions?